iothub/device/src/Transport/Amqp/AmqpConnectionHolder.cs

```diff
@@ -206,19 +206,17 @@ public async Task<AmqpIotConnection> EnsureConnectionAsync(TimeSpan timeout)
                     // Create AmqpConnection
                     amqpIotConnection = await _amqpIotConnector.OpenConnectionAsync(timeout).ConfigureAwait(false);
 
-                    if (_deviceIdentity.AuthenticationModel != AuthenticationModel.X509)
+                    if (_deviceIdentity.AuthenticationModel == AuthenticationModel.SasGrouped)
```

Comment:
_deviceIdentity.AuthenticationModel is an enum, so we can simplify this check

Reply:
What about AuthenticationModel.SasIndividual ? Does it not apply to that?

Reply:
for sas individual authenticated devices, the token refresher is associated with the amqp unit: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/iothub/device/src/Transport/Amqp/AmqpUnit.cs#L119

sas individual => amqp unit
sas group => amqp connection
x509 => no token refresher (and also, no multiplexing)